code/game/objects/items/tools/medical_tools.dm

```diff
@@ -0,0 +1,246 @@
+
+//CELLS here
+/obj/item/tool/geltool/gell  //Gell a CELL but GELL
+	name = "Gell classic vial"
+	desc = "A vial used to transport and store medical gel, NOT EDIBLE."
+	icon_state = "gell"
+	item_state = "gell"
+	icon = 'icons/obj/items/geltools.dmi'
+	var/gelcapacity = 840   //how much gel can you fit in this tool, THIS NUMBER WAS CHOSEN FOR ITS DIVISIBILITY
+	var/gelquantity = 840   //how much gel does it starts with
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/tool/geltool/gell/small
+	name = "Gell tiny vial"
+	desc = "A vial used to transport and store medical gel, NOT EDIBLE."
+	icon_state = "gelltiny"
+	item_state = "gelltiny"
+	gelcapacity = 280
+	gelquantity = 280
+	w_class = WEIGHT_CLASS_TINY
+
+
+/obj/item/tool/geltool/gell/attack(mob/living/M, mob/living/user)
+	if(M != user)
+		return ..()
+	if(gelquantity < 35)
```

Comment:
why 35? why not gelcapacity = the number of charges?

Reply:
Maybe add a message when gelquantity is low

Reply:
cause each "succ" consumes 35 charges so it needs to have those exact charges "or more"

Reply:
.... why would it not consumme 1 charge per succ?